doc/developer/building-frr-for-fedora.rst

```diff
@@ -98,6 +98,19 @@ And load the kernel modules on the running system:
 
    sudo modprobe mpls-router mpls-iptunnel
 
+Clear iptables
+^^^^^^^^^^^^^^
+
+Fedora ships with the ``firewalld`` service enabled. You may run into some
+issues with iptables rules it installs by default. If you wish to just stop
+the service and clear `ALL` rules do these commands:
```

Comment:
I'd make this a general note to check iptables rules to allow whatever protocol traffic you want, with a
```
.. note::

   Fedora ships with...
```
containing what you wrote here